homeassistant/components/huawei_lte/__init__.py

```diff
@@ -580,19 +536,6 @@ def service_handler(service: ServiceCall) -> None:
             schema=SERVICE_SCHEMA,
         )
 
-    for url, router_config in domain_config.items():
```

Comment:
We normally deprecate the domain key in the `CONFIG_SCHEMA` before removing the import and then remove the schema now or just keep the domain key in the schema marked with `cv.removed.